configs/tasks/vln_r2r.yaml

```diff
@@ -0,0 +1,31 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 500
+SIMULATOR:
+  AGENT_0:
+    SENSORS: ['RGB_SENSOR', 'DEPTH_SENSOR']
+  FORWARD_STEP_SIZE: 0.25
+  TURN_ANGLE: 15
+  HABITAT_SIM_V0:
+    GPU_DEVICE_ID: 0
+  RGB_SENSOR:
+    WIDTH: 256
+    HEIGHT: 256
+    HFOV: 45
+    TYPE: HabitatSimRGBSensor
+  DEPTH_SENSOR:
+    WIDTH: 256
+    HEIGHT: 256
+TASK:
+  TYPE: VLN-v0
+  SUCCESS_DISTANCE: 3.0
```

Comment:
The success distance is set to 3m here which is the distance defined in the original paper ([https://arxiv.org/abs/1711.07280](url)) and is used by most papers in this space.